docs/data-cookbooks/sns.md

```diff
@@ -68,15 +85,15 @@ sns:
         protocol: lambda
 ```
 
-Make sure that the receiver Lambda is configured in `lambdas.yml`.
+Make sure that the subscriber Lambda is configured in `lambdas.yml`. **Note that the Lambda name configured in `lambdas.yml`, `SnsS3Test`, needs to have its name postpended with `LambdaFunction` to have the ARN correctly found.**
 
-### SNS message format
+More information on configuring an SNS topic subscriptions in Cumulus can be found in our [developer documentation](../deployment/config_descriptions#sns).
 
-The `SfSnsReport` lambda receives the Cumulus message [(as the lambda's task input)](../workflows/input_output.html#2-resolve-task-input) and is responsible for publishing the message to the sftracker SNS Topic. But before it publishes the message, `SfSnsReport` makes a determiniation about the workflow status and adds an additional metadata key to the message at `message.meta.status`.
+### SNS message format
 
-First it determines whether the workflow has finished by looking for the `sfnEnd` key in the `config` object. If the workflow has finished, it checks to see if it has failed by searching the input message for a non-empty `exception` object. The lambda updates the `message.meta.status` with `failed` or `completed` based on that result. If the workflow is not finished the lambda sets `message.meta.status` to `running`.
+Subscribers to the SNS topics can expect to find the published message by parsing the JSON string representation of the message found in the [SNS event](https://docs.aws.amazon.com/lambda/latest/dg/eventsources.html#eventsources-sns) at `Records[].Sns.Message`. The value found at `Records[0].Sns.Message` will be a JSON stringified version of the ingest notification record for an execution, a granule, or a PDR.
```

Comment:
```suggestion
Subscribers to the SNS topics can expect to find the published message by parsing the JSON string representation of the message found in the [SNS event](https://docs.aws.amazon.com/lambda/latest/dg/eventsources.html#eventsources-sns) at `Records[0].Sns.Message`. The message will be a JSON stringified version of the ingest notification record for an execution, a granule, or a PDR.
```
Style suggestion - I don't believe there's ever a case where you'll have more than one Record (currently) in a SNS notification.